File: resources/js/types/malawian_status_type.ts
export type MalawianStatusType = 'Citizen' | 'Permanent Resident' | 'Work Permit' | 'Student Visa' | 'Visitor';

export const MALAWIAN_STATUS_TYPE_OPTIONS: { value: MalawianStatusType; label: string }[] = [
  { value: 'Citizen', label: 'Malawian Status Citizen' },
  { value: 'Permanent Resident', label: 'Malawian Status Permanent Resident' },
  { value: 'Work Permit', label: 'Malawian Status Work Permit' },
  { value: 'Student Visa', label: 'Malawian Status Student Visa' },
  { value: 'Visitor', label: 'Malawian Status Visitor' }
];

File: app/http/requests/malawian_status_type.go
package requests

type MalawianStatusType string

const (
	MalawianStatusCitizen           MalawianStatusType = "Citizen"
	MalawianStatusPermanentResident MalawianStatusType = "Permanent Resident"
	MalawianStatusWorkPermit        MalawianStatusType = "Work Permit"
	MalawianStatusStudentVisa       MalawianStatusType = "Student Visa"
	MalawianStatusVisitor           MalawianStatusType = "Visitor"
)
